modules/ipi-install-bmc-addressing-for-cisco-cimc.adoc

```diff
@@ -0,0 +1,63 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="bmc-addressing-for-cisco-cimc_{context}"]
+= BMC addressing for Cisco CIMC
+
+The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: <hostname>
+        role: <master | worker>
```

Comment:
While personally _I_, too, like the `|` usage, technically non-standard to guidelines and should probably be just something like:
```suggestion
        role: <type>
```
with a callout listing the valid values. I see there are a few other IPI-related instances in other modules, too, tho. Not blocking merge.